src/include/pbs_v1_module_common.i

```diff
@@ -0,0 +1,720 @@
+/*
```

Comment:
I think file should `.h` not `.i` as its header file. Also I would suggest filename to be like `pbs_module_common.h`

Reply:
@hirenvadalia this is not a header file really. In fact this contains defintions of functions..It is really the interface file (with stub implementations)

Reply:
@hirenvadalia: As @subhasisb stated, the file consists of global variable definitions and non-inline function definitions that really don't belong in a `.h` file so I used `.i` instead.  I included "v1" in the filename since it seemed specific to that version and to be consistent with `src/lib/Libpython/module_pbs_v1.c`.  I also included the version in `src/modules/python/pbs_v1_module_init.c`.  If you would still like the "v1" removed in either or both of those filenames, please let me know.

Reply:
@toonen thanks for explaining... I am good with keeping v1 in filename, if needed we can rename anytime in future.